.github/workflows/release-token-validation.yml

```diff
@@ -6,15 +6,13 @@ jobs:
   token-validation:
     runs-on: ubuntu-latest
     steps:
-      - name: Validate REPO_ACCESS_TOKEN 
+      - name: Validate REPO_ACCESS_TOKEN
         uses: actions/checkout@v2
         with:
           repository: '${{ github.repository_owner }}/twilio-oai'
           token:  ${{ secrets.REPO_ACCESS_TOKEN }}
       - name: Validate AWS tokens
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
-          aws-region: us-east-1
```

Comment:
From the [workflow log](https://github.com/kridai/twilio-cli/actions/runs/3940951808/jobs/6742713064#step:4:28), it seems the default region is set to 'ap-south-1'. Dont think its expected?